Makefile

```diff
@@ -430,16 +428,6 @@ DTS_OUT := $(TS_OUT_DIR)/corecrypto.d.ts
 # In release mode, fail if the lockfile does not match
 BUN_FROZEN_LOCKFILE := $(if $(RELEASE),--frozen-lockfile)
 
-# Generate WASM
-wasm-build-deps := $(RUST_SOURCES) $(FFI_LIBRARY)
-# Note the `&:` syntax: that's a "grouped target" rule: https://www.gnu.org/software/make/manual/html_node/Multiple-Targets.html
-$(JS_GEN) $(WASM_GEN) &: $(wasm-build-deps)
-	cd crypto-ffi/bindings/js && \
-	bun ubrn build web $(CARGO_BUILD_ARGS)
```

Comment:
Moving the ubrn step into the next stage feels like an error: now ubrn has to run even if only TS files have changed. That worsens the development experience. Is there some reason it had to be like that?

Reply:
You are right, there should be no reason to remove this step. I reverted that part.